base/tpl/posts-selector/post.html

```diff
@@ -1,6 +1,6 @@
 <div class="sow-post-selector-summary ui-draggable" data-id="<%= id %>">
 	<div class="sow-thumbnail">
-		<a href="<%= editUrl %>" target="_blank">
+		<a href="<%= editUrl %>" target="_blank" rel="noopener">
```

Comment:
I left this one out because I don't think we use these templates anymore. I'll log as an issue to check later.